autoload/lsp/omni.vim

```diff
@@ -182,14 +184,27 @@ function! lsp#omni#get_vim_completion_item(item, ...) abort
         let l:word = s:remove_typed_part(l:word)
     endif
     let l:kind = lsp#omni#get_kind_text(a:item)
+
+    " add user_data in completion item
+    if g:lsp_text_edit_enabled && has_key(a:item, 'textEdit')
+        let l:text_edit = a:item['textEdit']
+        let l:user_data = {
+                    \ 'type': s:completion_type,
+                    \ 'content': l:text_edit
+                    \ }
+    else
+        let l:user_data = {}
+    endif
+
     let l:completion = {
                 \ 'word': l:word,
                 \ 'abbr': l:abbr,
                 \ 'menu': '',
                 \ 'info': '',
                 \ 'icase': 1,
                 \ 'dup': 1,
-                \ 'kind': l:kind }
+                \ 'kind': l:kind,
+                \ 'user_data': string(l:user_data)}
```

Comment:
What happens to older vim that doesn't support `user_data`?

Reply:
I did not consider the old version.

I add [`has('patch-8.0.1493')`](https://github.com/vim/vim/commit/9b56a57cdae31f7a2c85d440392bf63d3253a158) check and only add `user_data` if supported this.

Reply:
Would be good to add it.

Reply:
Fixed  43a9752f1b0c6d0a395f43a54c9e4ec4ae713633 ([autoload/lsp/omni.vim#197](https://github.com/prabirshrestha/vim-lsp/pull/318/commits/43a9752f1b0c6d0a395f43a54c9e4ec4ae713633#diff-15712bb0a804586a2c6f928b17b68ce1L206)) and e8f3541741e27f101dbfae17e7a244f7bd037c64.